core/templates/core/panel.html

```diff
@@ -0,0 +1,63 @@
+{% extends 'core/base.html' %}
+
+{% block title %}Panel Gwardii{% endblock %}
+
+{% load list_events %}
+
+{% block content %}
+    <div class="absolute inset-y-0 left-0 flex items-center md:hidden h-fit">
+        <button id="mobile-menu-btn" type="button" class="relative inline-flex items-center justify-center rounded-md p-2 text-gray-400 hover:bg-gray-700 focus:bg-transparent" aria-controls="mobile-menu" aria-expanded="false">
+          <span class="absolute -inset-0.5"></span>
+          <span class="sr-only">Open sidebar</span>
+          <svg class="block h-6 w-6" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5" />
+          </svg>
+          <svg class="hidden h-6 w-6" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" aria-hidden="true">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M6 18L18 6M6 6l12 12" />
+          </svg>
+        </button>
+    </div>
+    <div class="flex flex-row nowrap flex-grow max-h-full overflow-auto">
+        <div id="sidebar" class="hidden fixed md:relative md:block min-w-72 max-w-72 h-full bg-gray-800 shadow-2xl overflow-auto">
+          <div class="flex flex-col gap-y-5 pb-36 md:pb-0" id="sidebar-content">
+              <div class="flex flex-grow flex-col justify-start divide-y divide-gray-600">
+                  <h3 class="flex mx-4 py-3 my-2 h-12 max-w-full items-center justify-start text-lg font-bold">Gwardia</h3>
+                  <a class="flex mx-4 py-3 my-2 h-12 max-w-full rounded-xl hover:bg-slate-600 items-center text-lg" href="{% url 'gwardia:index' %}">
```

Comment:
Generate links based on defined list in python. It's easier to provide
```py
{
  "name": "Spotkania"
  "url": "uwu"
  "icon": "fa-file-powerpoint"
}
```
Than copying everything...